ember-bootstrap/addon/components/bs-modal/footer.hbs

```diff
@@ -9,7 +8,7 @@
         <Button
           @type={{bs-default @submitButtonType "primary"}}
           @onClick={{@onSubmit}}
-          {{bs-conditional-attribute "disabled" @submitDisabled @submitDisabled}}
+          {{bs-conditional-attribute "disabled" @submitDisabled "true"}}
```

Comment:
MDN recommends using an empty string or the attribute's name for boolean attributes.

>  Boolean attributes are considered to be true if they're present on the element at all. You should set value to the empty string ("") or the attribute's name, with no leading or trailing whitespace.

I think we should follow that recommendation. Even though any string would work.

```suggestion
          {{bs-conditional-attribute "disabled" @submitDisabled "disabled"}}
```

Reply:
Ah, good point!